packages/framework/src/Model/SalesRepresentative/SalesRepresentative.php

```diff
@@ -130,4 +130,12 @@ public function getTelephone()
     {
         return $this->telephone;
     }
+
+    /**
+     * @return string
+     */
+    public function getFullName()
```

Comment:
ah, here is the method :ok_hand: you can add a return typehint here, btw

Reply:
added